homeassistant/components/geniushub/strings.json

```diff
@@ -4,24 +4,22 @@
       "user": {
         "title": "Genius Hub Configuration",
```

Comment:
Should this be "configuration" to be consistent with other changes?

Reply:
```suggestion
        "title": "Genius Hub configuration",
```